WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerMixpanel.m

```diff
@@ -27,6 +29,30 @@ - (void)beginSession
 {
     [Mixpanel sharedInstanceWithToken:[WordPressComApiCredentials mixpanelAPIToken]];
     [self refreshMetadata];
+    [self flagIfUserHasSeenLegacyEditor];
+}
+
+- (void)flagIfUserHasSeenLegacyEditor
+{
+    NSUserDefaults *standardDefaults = [NSUserDefaults standardUserDefaults];
+    if ([standardDefaults boolForKey:CheckedIfUserHasSeenLegacyEditor]) {
+        return;
+    }
+    
+    NSInteger sessionCount = [[[[Mixpanel sharedInstance] currentSuperProperties] numberForKey:@"session_count"] integerValue];
+    if ([[Mixpanel sharedInstance].currentSuperProperties[@"created_account_on_mobile"] boolValue]) {
```

Comment:
Similar to the comment above... this code is hard to read because it's basically a logic deduction without a label on it.  I'm a bit confused by the comment below because it seems not to match this line in my head (I'm not sure though, sorry about that... I'm just not familiar with the super property mentioned above).

One way in which we could make this code a bit easier to understand would be to replace:

```objective-c
if ([[Mixpanel sharedInstance].currentSuperProperties[@"created_account_on_mobile"] boolValue]) {
```

with something like

```objective-c
if ([self doesUserFulfillConditionX]) {
```

Or similar... where the line actually reports what we're attempting to verify, and the method definition shows the logic deduction we're using to reach that conclusion.